### DISCUSSION TITLE
Styling CSS classes with tailwind

### DISCUSSION THREAD

**Discussion by michael12987** [tailwindlabs/tailwindcss#10129] [Help]
Hi, i'm new to tailwind, i'm trying to style react component using tailwind, i can do it using css but i have no idea how to make it work using tailwind.

This is the component i'm trying to style: https://react-popup.elazizi.com/css-styling/#custom-css-classes
I guess it supposed to look something like this but again not sure what i'm doing .

```
`
export const PopupWin = () => (
  <NewPopup trigger={<button> Trigger</button>} position="right center">
    <div>Popup content here !!</div>
  </NewPopup>
);

const NewPopup = tw(Popup)`
popup-content{
    m-auto
    bg-black
    w-2/4
    p-1  
}
`;`

```
How it should be written?

**Comment by wongjn** [comment]
Try using an [arbitrary variant](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-variants) like `[&_.popup-content]`, for example, `[&_.popup-content]:m-auto [&_.popup-content]:bg-black [&_.popup-content]:w-2/4 [&_.popup-content]:p-1`

**Reply by michael12987** [reply]
Do you have an idea how can i write it in this form?

```
`const NewPopup = tw(Popup)`
popup-content{
    m-auto
    bg-black
    w-2/4
    p-1  
}
`;``
```

**Reply by wongjn** [reply]
No I don't – it seems like you may be after variant grouping but [here's a response to a suggestion for this syntax](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314

**Comment by Murphy624** [comment]
Code Error [Uspayserv](https://www.uspayserv.me/)
